tests/run-make/link-framework/Makefile

```diff
@@ -0,0 +1,23 @@
+# only-macos
```

Comment:
Does this test need to be written in Makefile? Can it be written in `rmake.rs`?

Reply:
Indeed - didn't know about `rmake.rs`, I think I made this PR before that was available